spec/models/maths_and_physics/school_eligibility_spec.rb

```diff
@@ -99,6 +134,11 @@
         context "and it doesn't have an education phase" do
           let(:school_attributes) { special_free_school_attributes.merge({phase: :not_applicable}) }
           it { is_expected.to be true }
+
+          context "and it is closed" do
+            let(:school_attributes) { special_free_school_attributes.merge({phase: :not_applicable, close_date: Date.yesterday}) }
+            it { is_expected.to be false }
+          end
```

Comment:
Do we need to test the closed context each time? I was wondering if operator precedence in the check method means this could be approached differently?

Reply:
Yeah that's a good point, I'll have a look at re-ordering them

Reply:
I've reordered the conditional to filter out closed schools earlier, I've also reduced the number of contexts down